### PR TITLE
fix: api quick start guide

### PR DESCRIPTION
**Pull request by lakhansamani**
* Fix php bulk request example

### CODE REVIEW

File: content/api/javascript/apireference.md [hunk 1]
```diff
@@ -322,24 +318,31 @@ appbaseRef
...
 appbaseRef
-	.reactiveSearchv3([{
-		id: "book_search",
-		dataField: ["original_title"],
-		size: 10,
-		value: "harry"
-	}], {
-		userId: "jon@appbase.io"
-	})
+	.reactiveSearchv3(
```
Comment (siddharthlatest) [inline content/api/javascript/apireference.md:333]: @lakhansamani This looks like an unintended side-effect.

File: content/api/javascript/apireference.md [hunk 2]
```diff
@@ -523,19 +533,19 @@ Continuously stream results of search query on a given `type` to a URL. **search
 ```js
 appbaseRef.searchStreamToURL(
```
Comment (siddharthlatest) [inline content/api/javascript/apireference.md:534]: @lakhansamani This as well.

File: content/api/javascript/quickstart.md [hunk 3]
```diff
@@ -122,41 +128,39 @@ Now that we are able to store data, let's try to get the data back from [appbase
 
 ```js
 appbaseRef.get({
-	type: "books",
-	id: "X1"
+ type: "_doc",
```
Comment (siddharthlatest) [inline content/api/javascript/quickstart.md:131]: @lakhansamani This as well.

File: content/api/javascript/quickstart.md [hunk 4]
```diff
@@ -122,41 +128,39 @@ Now that we are able to store data, let's try to get the data back from [appbase
...
-	"_version": 5,
-	"found": true,
-	"_source": {
-		"department_name": "Books",
-		"department_name_analyzed": "Books",
-		"department_id": 1,
-		"name": "A Fake Book on Network Routing",
-		"price": 5595
-	}
+ "_index": "good-books-demo",
```
Comment (siddharthlatest) [inline content/api/javascript/quickstart.md:141]: See this too.

File: content/api/rest/quickstart.md [hunk 5]
```diff
@@ -12,170 +12,142 @@ sidebar: 'api-reference'
 
 This is a quick start guide to working with the [appbase.io REST API](https://rest.appbase.io).
 
-## Creating an App
+## Create Cluster
 
-This gif shows how to create an app on appbase.io, which we will need for this quickstart guide.
+You can run dedicated Elasticsearch clusters with Appbase.io services to meet your search requirements using [Appbase.io clusters](/docs/hosting/clusters/).
```
Comment (siddharthlatest) [inline content/api/rest/quickstart.md]: @lakhansamani This needs an edit imo. We should cover all options. Text can be:

You can start by creating an [Elasticsearch cluster with appbase.io](href1) or [bring your Elasticsearch cluster](href2) or [self-host appbase.io](href3).

File: content/api/rest/quickstart.md [hunk 6]
```diff
@@ -12,170 +12,142 @@ sidebar: 'api-reference'
...
 
-![](https://i.imgur.com/r6hWKAG.gif")
+-   Log in to[Appbase Dashboard](https://dashboard.appbase.io), and create a new cluster.
+-   Copy the URL of your cluster for further actions
 
-Log in to <span class="fa fa-external-link"></span> [Appbase Dashboard](https://appbase.io/scalr/), and create a new app.
+You can read more about how to [create cluster](/docs/hosting/clusters/) and its [pricing](https://appbase.io/pricing/).
```
Comment (siddharthlatest) [inline content/api/rest/quickstart.md]:
```suggestion
You can read see the different options for creating a cluster and their pricing over [here](https://appbase.io/pricing/).
```

File: content/api/javascript/quickstart.md [hunk 7]
```diff
@@ -24,19 +24,26 @@ It can't:
 
 [Appbase.io - the database service](https://appbase.io) is opinionated about cluster setup and hence doesn't support the Elasticsearch devops APIs. See [rest.appbase.io](https://rest.appbase.io) for a full reference on the supported APIs.
 
-This is a quick start guide to whet the appetite with the possibilities of data streams.
+## Create Cluster OR Use Existing Elasticsearch
 
-## Creating an App
+You can run dedicated Elasticsearch clusters with Appbase.io services to meet your search requirements using [Appbase.io clusters](/docs/hosting/clusters/).
```
Comment (siddharthlatest) [inline content/api/javascript/quickstart.md]: @lakhansamani Refer to the comment in the Rest quickstart for the exact text to use here.

File: content/api/javascript/quickstart.md [hunk 8]
```diff
@@ -24,19 +24,26 @@ It can't:
 
 [Appbase.io - the database service](https://appbase.io) is opinionated about cluster setup and hence doesn't support the Elasticsearch devops APIs. See [rest.appbase.io](https://rest.appbase.io) for a full reference on the supported APIs.
 
-This is a quick start guide to whet the appetite with the possibilities of data streams.
+## Create Cluster OR Use Existing Elasticsearch
```
Comment (siddharthlatest) [inline content/api/javascript/quickstart.md]:
```suggestion
## Create Cluster
```

**Review by siddharthlatest — CHANGES_REQUESTED** [review]
For the indent issues, these are just questions. If the indent looks good, you can resolve them.

I've made requested some content changes and left some other suggestions.

Overall, this is looking good.

File: content/api/rest/quickstart.md (hunk 5 above)
Comment (siddharthlatest) [reply content/api/rest/quickstart.md]: hrefs need to point to the right pages/sections.

File: content/api/javascript/apireference.md (hunk 1 above)
Comment (lakhansamani) [reply content/api/javascript/apireference.md:333]: @siddharthlatest it was unintended earlier, it looks correct now. Verified in preview.

**Pull request merged**